src/main/java/com/rarchives/ripme/ripper/rippers/ErotivRipper.java

```diff
@@ -0,0 +1,94 @@
+package com.rarchives.ripme.ripper.rippers;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.jsoup.Connection.Response;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+import com.rarchives.ripme.ripper.AbstractHTMLRipper;
+import com.rarchives.ripme.utils.Http;
+
+/**
+ *
+ * @author randomcommitter
+ */
+public class ErotivRipper extends AbstractHTMLRipper {
+
+    boolean rippingProfile;
+
+
+    public ErotivRipper (URL url) throws IOException {
+        super(url);
+    }
+
+    @Override
+    public String getDomain() {
+            return "erotiv.io";
+    }
+
+    @Override
+    public String getHost() {
+            return "erotiv";
+    }
+
+    @Override
+    public String getGID(URL url) throws MalformedURLException {
+        Pattern p = Pattern.compile("^https?://erotiv.io/e/([0-9]*)/?$");
```

Comment:
IMO you should always account for potential `www.`.

Reply:
@rautamiekka fixed! Thanks for the tip